include/picongpu/particles/ionization/byField/ADK/ADK.def

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2015-2020 Marco Garten
+/* Copyright 2015-2020 Marco Garten and Jakob Trojok
```

Comment:
Please use comma instead of `and` in such file headers (it is in a few in this PR) to be consistent with the other files.